tests/integration/test_agent.py

```diff
@@ -16,17 +15,6 @@
 
 workspace_base = os.getenv('WORKSPACE_BASE')
 
-# make sure we're testing in the same folder of an existing config.toml
-if os.path.exists('config.toml'):
```

Comment:
@tobitege I removed your hack here since it assumes `config.toml` holds the canonical value. If you still see the wrong workspace path issue after setting `persist_sandbox = false`, please open up a new issue.

Reply:
Ok 👍